rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/nodesource/serialization/NodeSourceConfigurationParser.java

```diff
@@ -75,7 +77,7 @@ public NodeSourceConfiguration parseNodeSourceConfiguration(String json) {
                                                infrastructurePluginDescriptor,
                                                policyPluginDescriptor);
         } catch (RuntimeException e) {
-            throw new IllegalArgumentException("The imported node source has incorrect parameters.", e);
+            throw new IllegalArgumentException("The imported file has incorrect parameters.", e);
         }
     }
 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "infrastructurePluginDescriptor" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)